src/commands/dora/README.md

```diff
@@ -0,0 +1,90 @@
+# DORA Metrics
+
+Send deployment events for DORA Metrics from CI.
+
+## Usage
+
+### Commands
+
+#### `deployment`
+
+**Warning:** The `dora deployment` command is in beta. It requires you to set `DD_BETA_COMMANDS_ENABLED=1`.
+
+This command sends details to Datadog about a deployment of a `service`.
+
+```bash
+$ DD_BETA_COMMANDS_ENABLED=1 datadog-ci dora deployment [--service #0] [--env #0] [--dry-run]
+
+━━━ Options ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  --started-at #0            In Unix seconds or ISO8601. (Examples: 1699960648, 2023-11-14T11:17:28Z)
+  --finished-at #0           In Unix seconds or ISO8601. (Examples: 1699961048, 2023-11-14T11:24:08Z)
+  --git-repository-url #0    Example: https://github.com/DataDog/datadog-ci.git
+  --git-commit-sha #0        Example: 102836a25f5477e571c73d489b3f0f183687068e
+  --skip-git                 Disables git info attributes. Change Lead Time will not be available
+```
+
+For example:
+
+```bash
+export DD_BETA_COMMANDS_ENABLED=1
+
+datadog-ci dora deployment --service my-service --env prod \
+    --started-at 1699960648 --finished-at 1699961048 \
+    --git-repository-url https://github.com/my-organization/my-repository \
+    --git-commit-sha 102836a25f5477e571c73d489b3f0f183687068e
+```
+
+- `--service` (or `DD_SERVICE`) and `--started-at` are always required and `--git-repository-url` and `--git-commit-sha` are necessary for Change Lead Time.
```

Comment:
isn't this line redundant with the lines below? We have a (required) note and we mention what is necessary for change lead time